### PR TITLE
ci-operator: add e2e job for crun-wasm

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

File: ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml [hunk 1]
```diff
@@ -264,6 +264,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (haircommander) [inline ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml:268]: I don't think we need a presubmit to always run. I think a periodic is sufficient then we can run the presubmit if needed

Comment (sohankunkerkar) [reply ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml]: Fixed

**Review by mrunalp — APPROVED** [review]

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Pull request merged**